alb/alb.go

```diff
@@ -20,7 +20,10 @@ import (
 )
 
 // New creates a controller.Updater for attaching to ALB target groups on first update.
-func New(region string, targetGroupNames []string, targetGroupDeregistrationDelay time.Duration) controller.Updater {
+func New(region string, targetGroupNames []string, targetGroupDeregistrationDelay time.Duration) (controller.Updater, error) {
+	if len(targetGroupNames) == 0 {
+		return nil, fmt.Errorf("Unable to create Alb Updater: missing target group names")
```

Comment:
Errors should always be lowercase. Also use `error.New` if you don't need to format a string.